docker-compose.yml

```diff
@@ -49,6 +49,8 @@ services:
     volumes:
       - hsmtmpdir:/tmp/
       - hsmconfig:/app/
+    depends_on:
+      - db
```

Comment:
:+1: 

does this let us drop the wait-for-it scripts?

Reply:
I don't think so. It'll just start the container after the DB, but provisioning might take time, so the `wait-for-it` script is still needed.